Makefile.inc1

```diff
@@ -1036,12 +1036,16 @@ WORLDTMP_MTREEFLAGS=	-deUW
 # that are created by mtree to be owned by root/wheel.
 DESTDIR_MTREEFLAGS+=	-W
 .endif
-DISTR_MTREE=	${MTREE_CMD}
+.if defined(DB_FROM_SRC)
+DISTR_MTREEFLAGS=	-N ${.CURDIR}/etc
+.endif
+DISTR_MTREECMD=	${MTREE_CMD}
 .if ${BUILD_WITH_STRICT_TMPPATH} != 0
-DISTR_MTREE=	${WORLDTMP}/legacy/usr/sbin/mtree
+DISTR_MTREECMD=	${WORLDTMP}/legacy/usr/sbin/mtree
 .endif
```

Comment:
I might have just used a += on DISTR_MTREE moving it after the BUILD_WITH_STRICT_TMPPATH as the diff would be slightly smaller, but this is fine.

Reply:
That breaks the other _MTREE variables defined below, no? They’re not :=

Reply:
Or at least it changes them, whether it matters, and whether they already have -N in that case, I don’t know